hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/table/action/cluster/strategy/TestSparkClusteringPlanPartitionFilter.java

```diff
@@ -53,9 +53,9 @@ public void setUp() {
   @Test
   public void testFilterPartitionNoFilter() {
     HoodieWriteConfig config = hoodieWriteConfigBuilder.withClusteringConfig(HoodieClusteringConfig.newBuilder()
-            .withClusteringPlanPartitionFilterMode(ClusteringPlanPartitionFilterMode.NONE)
-            .build())
-        .build();
+                    .withClusteringPlanPartitionFilterMode(ClusteringPlanPartitionFilterMode.NONE)
```

Comment:
Please revert this change because there is no changes.

Reply:
done